src/System.Reflection/tests/AssemblyTests.cs

```diff
@@ -465,6 +465,15 @@ public void LoadWithPartialName()
             string simpleName = typeof(AssemblyTests).Assembly.GetName().Name;
             var assembly = Assembly.LoadWithPartialName(simpleName);
             Assert.Equal(typeof(AssemblyTests).Assembly, assembly);
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Assembly.LoadFromWithPartialName() not supported on UapAot")]
+        public void LoadWithPartialName_Neg()
+        {
+            Assert.Throws<ArgumentNullException>(() => Assembly.LoadWithPartialName(null));
+            Assert.Throws<ArgumentException>(() => Assembly.LoadWithPartialName(""));
```

Comment:
Is a parameter name included in these exceptions?  If so, can we validate it?

Reply:
They do and the one from `LoadWithPartialName` is wrong. 👍